src/models/event.js

```diff
@@ -299,7 +310,16 @@ utils.extend(module.exports.MatrixEvent.prototype, {
     },
 
     /**
-     * Attempt to decrypt this event.
+     * Check if this event is currently being decrypted.
+     *
+     * @return {boolean} True if this event is currently being decrypted, else false.
+     */
+    isBeingDecrypted: function() {
+        return this._decrypting;
+    },
+
+    /**
+     * Start the process of trying to decrypt this event.
```

Comment:
oh, another reason, which I'll put down here for future reference:

In the situation that we already have an attempt ongoing, we effectively queue up a retry in the event that decryption fails, and return immediately. If we returned a promise here, we'd want to resolve that promise once the *retry* completes - which is a slightly complicated dance which seems over-engineered when we never use the promise.